client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/style.scss

```diff
@@ -270,6 +270,22 @@ $design-button-primary-color: rgb( 17, 122, 201 );
 			}
 		}
 	}
+
+	.step-container__header .formatted-header {
+		h1.formatted-header__title {
```

Comment:
Hmmm...I think this made the styles of the title of the v13n design picker weird

| Before | After |
| - | - |
| ![image](https://user-images.githubusercontent.com/13596067/176165344-13113084-f691-4f1c-8321-4f4f8a653f0e.png) | ![image](https://user-images.githubusercontent.com/13596067/176165384-ec753192-810c-4bcb-84fd-e119734bd1f2.png) |

Reply:
Working on a fix! Thanks!

Reply:
@arthur791004 , there's a PR to fix this [here](https://github.com/Automattic/wp-calypso/pull/65047) 👍

Reply:
thanks 💯